change/@ni-nimble-components-3b2df972-f5ae-4415-be0f-8e4268970423.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Think you mentioned it but the snapshots looked snipped off, are these stories too large now?
![image](https://github.com/user-attachments/assets/bd7a89c9-d99c-4690-9cc6-e247ec870b96)

Reply:
This is actually a different problem that the bottom margin of the control wasn't large enough to account for error text possibly being below the bottom row of controls. This isn't a problem any more with the way I'm planning to reorder the matrix stories (i.e. the bottom row shouldn't have error text anymore), but I've changed the margin to be more robust.